CMakeLists.txt

```diff
@@ -343,3 +342,13 @@ INSTALL(
     DESTINATION CMake
     COMPONENT dev
 )
+
+#Recursively list of all headers, with paths relative to include/tins
+FILE(GLOB_RECURSE INCLUDE_FILES RELATIVE include/tins "*.h")
```

Comment:
Does the `RELATIVE` clause restrict the lookup to the `include/tins` directory or will this search for any `.h` files starting from the project's root?

Reply:
Indeed, it's project root, expression should now be "include/tins/*.h".

Reply:
Yeah, that makes sense. Otherwise this will probably end up installing the googlemock headers globally.

Reply:
Have a look now, narrowed the scope and simplified.